app/javascript/controllers/actors_controller.js

```diff
@@ -0,0 +1,40 @@
+import { Controller } from 'stimulus'
+import axios from 'axios'
+import { csrfToken, jumpToCreator } from './stimulus_modules'
+
+export default class extends Controller {
+  // Intercept Rails UJS's ajax:success callback and update the modal with the resulting content.
+  // This avoids having to have a separate .js.erb partial in a typical Rails AJAX setup.
+  success (event) {
+    const [data, status, xhr] = event.detail // eslint-disable-line no-unused-vars
+    document.getElementsByClassName('modal-content').item(0).innerHTML = data.body.innerHTML
+  }
+
+  append (event) {
+    event.preventDefault()
+    const data = { actor_id: event.target.dataset.id }
+    data.index = document.getElementsByClassName('nested-fields').length
```

Comment:
There might be more than one set of nested-fields on the page in the future. In Christie's mockups, Creators, Collaborators, and Acknowledgements are all on the same tab (and might have nested fields). Probably a good idea to scope this to within `#creator_aliases`